src/__stories__/feedback-screen-story.tsx

```diff
@@ -1,66 +1,24 @@
 import * as React from 'react';
-import {
-    ErrorFeedbackScreen,
-    InfoFeedbackScreen,
-    SuccessFeedbackScreen,
-    FeedbackScreen,
-} from '../feedback-screen';
+import {FeedbackScreen} from '../feedback-screen';
 import {ButtonLink, ButtonPrimary, ButtonSecondary} from '../button';
 import Stack from '../stack';
 import {ThemeVariant, useIsInverseVariant} from '../theme-variant-context';
 import {useTheme} from '../hooks';
 import {Box, FixedToTop, Text3, Text2} from '..';
 
 export default {
-    title: 'Components/Screens/FeedbackScreen',
+    title: 'Components/Screens/FeedbackScreen/FeedbackScreen',
```

Comment:
is the last `/FeedbackScreen` needed?

Reply:
no! I had some problems with the url during the sepparation of stories and I must have forgotten it. Thanks!